packages/ilios-common/addon/components/course/manage-objective-parents.hbs

```diff
@@ -34,8 +34,14 @@
           >
             <h4
               class="competency-title"
+              data-test-competency-title
             >
               {{competency.title}}
+              {{#if competency.parent}}
```

Comment:
Competencies can be arbitrarily deep, is this the parent or the domain? If the domain that will need to be fetched with `getDomain` in case it isn't the direct parent.

Reply:
domain/competencies only go one level deep, per business rules. 
the database schema technically supports arbitrary nesting levels, but the application code won't support it.
if there's a parent, then it must be a top level domain.

Reply:
✅